apps/apps_publisher.py

```diff
@@ -50,9 +51,12 @@ def __tag(self, app: ComposeApps.App, tag: str):
                                 .format(app.name, service_name))
             image_url = expandvars(image_url_template)
             logger.info('Service image url: %s', image_url)
-            if not image_url.startswith(self._image_base_url):
-                # TODO: verify if URL points to some other factory
+            if not image_url.startswith(self._registry_host):
```

Comment:
How do you refer to dockerhub images after this change?

Reply:
As usually, the change doesn't impact it. The given business logic tags only URLs of images that are hosted in Foundries Docker registry, if an image URL refers to some 3rd party registry/hub then we don't need to do anything, so the code just call 'continue' in this case.

Reply:
ok, so if I declare it as:
```
image: foo:latest
```
it will default to dockerhub and it will still work?

Reply:
yes, provided that 'foo:latest' is hosted in the docker hub.